utils/node_local_test.py

```diff
@@ -1449,6 +1451,35 @@ def check_usage(self, inodes=None, open_files=None, pools=None, containers=None)
             assert rc.json['response']['containers'] == containers, rc
         return rc.json['response']
 
+    def _evict_path(self, path):
+        """Evict a path from dfuse"""
+        cmd = ['filesystem', 'evict', path]
+        rc = run_daos_cmd(self.conf, cmd, use_json=True)
+        print(rc)
+        assert rc.returncode == 0
+
+        return rc.json['response']
+
+    def evict_and_wait(self, paths):
+        """Evict a number of paths from dfuse"""
+        inodes = []
+        for path in paths:
+            rc = self._evict_path(path)
+            inodes.append(rc['inode'])
+
+        sleeps = 0
+
+        for inode in inodes:
+            found = True
+            while found:
+                rc = self.check_usage(inode)
+                print(rc)
+                found = rc['resident']
+                if not found:
+                    time.sleep(1)
+                    sleeps += 1
+                    assert sleeps < 10, 'Path still present 10 seconds after eviction'
```

Comment:
1. Shouldn't sleeps be reset to 0 for each inode?
2. Maybe move the assert before `time.sleep`? Otherwise we sleep for 1 second, don't check the path again, and then assert

Reply:
1, no, this is a lazy way of doing a timeout.
2, yes, the actual sleep should be last here.  Will fix.

Reply:
1. So ~10 seconds total for all paths?